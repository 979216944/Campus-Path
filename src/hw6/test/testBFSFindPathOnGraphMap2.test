# Test BFS finding a shortest path.

# Load graph
LoadGraph map2 map2.tsv

# Find a shortest path from location1 to location7 and the reverse
FindPath map2 location1 location7
FindPath map2 location7 location1