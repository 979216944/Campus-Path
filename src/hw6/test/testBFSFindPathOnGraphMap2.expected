# Test BFS finding a shortest path.

# Load graph
loaded graph map2

# Find a shortest path from location1 to location7 and the reverse
path from location1 to location7:
location1 to location8 via road1
location8 to location7 via road7
path from location7 to location1:
location7 to location8 via road7
location8 to location1 via road1